src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterGood.java

```diff
@@ -3,9 +3,9 @@
 /** some javadoc. */
 public class InputWhitespaceAfterGood {
 
-  int xyz;       // multiple space between content and double slash.
+  int xyz; // multiple space between content and double slash.
   int abc; //       multiple space between double slash and comment's text.
-  int pqr;       //     testing both.
+  int pqr; //     testing both.
```

Comment:
we lost this edge case after formatting. This edge case was explicitly mentioned in google style guide doc:

[4.6.2 Horizontal whitespace](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace)

> 6. Between any content and a double slash (//) which begins a comment. Multiple spaces are allowed.
> 7. Between a double slash (//) which begins a comment and the comment's text. Multiple spaces are allowed.

Reply:
should we revert this changes or not? @romani

Reply:
Please revert this changes.
We need this violations.

We will use what formatter wants in InputFormatted.

Reply:
if this the case then this section will be completely excluded